# HG changeset patch
# User Craig Matsuura <cmatsuura@vivnt.com>
# Date 1407992571 21600
# Node ID 6b73a9da7ca75c9aa4a965bd6ed492c7fb4d481c
# Parent  f708b371b829d17133a201340eadad1faadf576f
Fixed to build, used changes found in pulseaudio-2.0

diff --git a/src/daemon/dumpmodules.c b/src/daemon/dumpmodules.c
--- a/src/daemon/dumpmodules.c
+++ b/src/daemon/dumpmodules.c
@@ -89,6 +89,10 @@
     }
 }
 
+#ifndef OS_IS_WIN32
+extern const lt_dlsymlist lt_preloaded_symbols[];
+#endif
+
 static int is_preloaded(const char *name) {
     const lt_dlsymlist *l;
 
